code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -229,19 +241,22 @@ namespace WijmoProvider.Feature {
         /**
          * Triggers the events of OnCellValueChange associated to a specific column in OS
          * @param rowNumber Number of the row to trigger the events
-         * @param binding Binding of the column that contains the associated events
+         * @param columnUniqueID Id of the Column that contains the associated events
          * @param oldValue Value of the cell before its value has changed (Old)
          * @param newValue Value of the cell after its value has changed (New)
          */
         private _triggerEventsFromColumn(
             rowNumber: number,
-            binding: string,
+            columnUniqueID: string,
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             oldValue: any,
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             newValue: any
         ) {
-            const column = this._grid.getColumn(binding);
+            const column = this._grid
```

Comment:
We should use: 
const column = this._grid.getColumn(columnUniqueID);

Reply:
done